diff --git a/modules/fcgid/fcgid_conf.h b/modules/fcgid/fcgid_conf.h
index 47d68bb..44da00e 100644
--- a/modules/fcgid/fcgid_conf.h
+++ b/modules/fcgid/fcgid_conf.h
@@ -139,7 +139,7 @@ typedef struct {
  */
 #define INITENV_KEY_LEN 64
 #define INITENV_VAL_LEN 256
-#define INITENV_CNT 64
+#define INITENV_CNT 1024
 typedef struct {
     char initenv_key[INITENV_CNT][INITENV_KEY_LEN];
     char initenv_val[INITENV_CNT][INITENV_VAL_LEN];
